.changeset/chilly-oranges-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-avatax": minor
```

Comment:
suggestion: imho this is a breaking change and we perhaps should make it as major version bump

Reply:
It doesn't change anything because it will be auto-deployed anyway. We would need to have a separate manifest and upgrade system for that

Reply:
Agree this may be a breaking behavior, but first 20 chars probably were not used anywhere anyway (how?), so I think we rather changed something from unusable to usable

Reply:
I will merge it and let client know that we changed what we pass into document code